public/index.html

```diff
@@ -2,15 +2,35 @@
 <html lang="cs">
 	<head>
 		<meta charset="utf-8" />
-		<link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
 		<meta name="viewport" content="width=device-width, initial-scale=1" />
 		<meta name="theme-color" content="#264653" />
 		<meta
 			name="description"
 			content="Web application that connects programmers based on their experience, interests and the type of person they are looking for (mentor, buddy, mentee) through matching and contact sharing."
 		/>
-		<!-- <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" /> -->
-		<link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+		<!-- favicon -->
+		<link
+			rel="apple-touch-icon"
+			sizes="180x180"
+			href="/apple-touch-icon.png"
```

Comment:
I would use the `%PUBLIC_URL%` here. :wink:
https://create-react-app.dev/docs/using-the-public-folder/
```suggestion
			href="%PUBLIC_URL%/apple-touch-icon.png"
```